configs/mae/upernet_mae_base_12_512_slide_160k_ade20k_pt2ft.py

```diff
@@ -0,0 +1,50 @@
+_base_ = [
```

Comment:
configs/mae/upernet_mae_base_12_512_slide_160k_ade20k_pt2ft.py can be changed to configs/mae/upernet_mae-base_fp16_8x2_512x512_160k_ade20k.py